core/db/migrate/20161006211441_create_spree_store_credit_ledger_entry.rb

```diff
@@ -0,0 +1,9 @@
+class CreateSpreeStoreCreditLedgerEntry < ActiveRecord::Migration[5.0]
+  def change
+    create_table :spree_store_credit_ledger_entries do |t|
```

Comment:
I would also like to see a source here in order to track where the action came from (for example, a payment or admin user).